tsconfig.json

```diff
@@ -22,6 +22,6 @@
       "@/*": ["./*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "app/login/page.tsx", "hooks/useUser.ts"],
```

Comment:
Please, check if it's really needed and for what it is used

Reply:
I found we don't actually need it. This line is indicates which files should be compiled to js. So it seems repetitive to me, since **/*.ts and **/*.tsx already include all ts and tsx files to compilation. Just corrected it.